### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by bahoz-ali**
- adding contact form section
- giving style to contact form.
- validation of the form.
- collecting data by using **https://formspree.io**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -289,5 +289,61 @@ <h1 class="title">Skills</h1>
...
+        <div class="contact__header">
+          <h1 class="header__title">Contact Me</h1>
+          <p class="header__text">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <form
```
Comment (SajjadAhmad14) [inline index.html:305]: The form does not have any styling and does not follow the design. Kindly follow the Figma design which includes color, background images, button hover effects, etc.

Yours:
yours-form

Design:
form1
form2

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Bahoz, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -289,5 +289,61 @@ <h1 class="title">Skills</h1>
...
+          class="contact-form"
```
Comment (tufoinnkuo10) [inline index.html:306]: - [ optional] you can display your form flex to layout elements

File: index.html (hunk 1 above)
Comment (tufoinnkuo10) [reply index.html:305]: - [ opional] next time please try to start styling normally with the mobile version and only include media query when switching to the desktop version.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @bahoz-ali ,  

Your project is complete! Great job For making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (bahoz-ali) [reply index.html:306]: thanks, I fixed that.

**Pull request merged**